userservice/src/main/java/com/infodation/userservice/services/UserServiceImpl.java

```diff
@@ -46,12 +50,14 @@ public Page<User> getAll(Pageable pageable, String name) {
     }
 
     @Override
+    @Cacheable(value = "users", key = "#userId")
```

Comment:
Can you help show me the value in Redis to make sure the data has been saved into Redis ?